lms/security.py

```diff
@@ -238,6 +239,9 @@ def get_lti_user(request) -> Optional[LTIUser]:
         # LTI users and the corresponding user in H
         request.find_service(UserService).upsert_user(lti_user)
 
+        # Attach useful information to sentry in case we get an exception further down the line
+        set_tag("application_instance_id", lti_user.application_instance_id)
```

Comment:
We could also send the organization ID but that's more awkward to do right now. Let's start with just the AI.id